$NetBSD: patch-CMakeLists.txt,v 1.1 2017/09/26 17:38:22 minskim Exp $

Don't force -std=gnu99 if we want to possibly use C11
for aligned_alloc

--- CMakeLists.txt.orig	2017-07-19 08:08:12.000000000 +0000
+++ CMakeLists.txt
@@ -201,8 +201,6 @@ if(CMAKE_SYSTEM_PROCESSOR STREQUAL i386 
    CMAKE_SYSTEM_PROCESSOR STREQUAL amd64 OR
    CMAKE_SYSTEM_PROCESSOR STREQUAL AMD64)
     if(CMAKE_C_COMPILER_ID STREQUAL GNU)
-        # We need C99 (GNU99 more exactly)
-        SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=gnu99")
         set(COMPILER_SUPPORT_SSE2 TRUE)
         if(CMAKE_C_COMPILER_VERSION VERSION_GREATER 4.7 OR CMAKE_C_COMPILER_VERSION VERSION_EQUAL 4.7)
             set(COMPILER_SUPPORT_AVX2 TRUE)
